test/jdk/valhalla/valuetypes/NullRestrictedArraysTest.java

```diff
@@ -25,8 +25,7 @@
 /*
  * @test
  * @enablePreview
- * @run junit/othervm NullRestrictedArraysTest
- * @run junit/othervm -XX:-UseArrayFlattening NullRestrictedArraysTest
```

Comment:
Why remove the no flattening configurations?

Reply:
If array flattening is disabled, but field flattening is enabled, we might run into situation where the new flat array primitive fails -- that primitive requires same level of flattening for both arrays and fields.

Reply:
I've now re-enabled this combo -- but disabled both array flattening and nullable flattening.